astroid/tests/unittest_brain_numpy_core_function_base.py

```diff
@@ -0,0 +1,86 @@
+# -*- encoding=utf-8 -*-
+# Copyright (c) 2017-2018 hippo91 <guillaume.peillex@gmail.com>
+
+# Licensed under the LGPL: https://www.gnu.org/licenses/old-licenses/lgpl-2.1.en.html
+# For details: https://github.com/PyCQA/astroid/blob/master/COPYING.LESSER
+import unittest
+import contextlib
+
+try:
+    import numpy  # pylint: disable=unused-import
+
+    HAS_NUMPY = True
+except ImportError:
+    HAS_NUMPY = False
+
+from astroid import builder
+
+
+class SubTestWrapper(unittest.TestCase):
```

Comment:
Even if we can't `subTest` right now (check my previous question on this topic), we should move these definitions into a common test file and use them from there.

Reply:
All occurrences of this mock have been deleted.